### PR TITLE
First pass on new technical issue template

### PR DESCRIPTION
**Pull request by HLCharlie**
I am using the old technical issue file, renaming it, and changing the contents. Will also delete the other old files in ISSUE_TEMPLATES.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Standard Technical Issue.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Description
```
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard Technical Issue.md:1]: Can we change the name of this to not have spaces in it? spaces almost always end up causing a bunch of issues down the line. Also for the description on the issue, there' s a solution but nothing about what it's solving.

Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard Technical Issue.md]: renamed

File: .github/ISSUE_TEMPLATE/Standard-Technical-Issue.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## Description
+
+< Description of the problem. >
+
+## Negative Impact:
+
+< what problems does, will, or can the problem described cause for Dev, QA, or DevOps. >
```
Comment (Chekote) [inline .github/ISSUE_TEMPLATE/Standard-Technical-Issue.md:7]: The English here is a little off

```suggestion
< How the described problem does, will, or can affect Dev, QA, or DevOps. >
```

File: .github/ISSUE_TEMPLATE/Standard-Technical-Issue.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Proposed Solution:
+
+< Describe the propsed solution for the problem. >
```
Comment (Chekote) [inline .github/ISSUE_TEMPLATE/Standard-Technical-Issue.md:11]: Typo

```suggestion
< Describe the proposed solution for the problem. >
```

**Pull request merged**